steps/tryoutapps-shaman/1.0.2/step.yml

```diff
@@ -0,0 +1,72 @@
+title: TryoutApps Shaman
+summary: |
+  Uploads apps to Infinum's TryoutApps
+description: |
+  Uploads apps to Infinum's TryoutApps using shaman deploy script.
+  Only one build at the time can be uploaded.
+website: https://github.com/infinum/TryoutApps-Shaman-Bitrise-Deploy
+source_code_url: https://github.com/infinum/TryoutApps-Shaman-Bitrise-Deploy
+support_url: https://github.com/infinum/TryoutApps-Shaman-Bitrise-Deploy/issues
+published_at: 2019-11-14T17:05:58.305178+01:00
+source:
+  git: https://github.com/infinum/TryoutApps-Shaman-Bitrise-Deploy.git
+  commit: a411fbae40792a30fef3996811744432053aad4f
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
```

Comment:
Invalid type tags. 
> Failed, error:
   - invalid type_tag: ios
   - invalid type_tag: macos
   - invalid type_tag: android

See https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md:
```
Available project_type_tags:

    ios
    macos
    android
    xamarin
    react-native
    cordova
    ionic
    flutter
    web

If step is available for all project types, do not specify project_type_tags, otherwise specify every project types, with which the step can work.

type_tags are used to categorise the steps based on it's functionality.

Available type_tags:

    access-control
    artifact-info
    installer
    deploy
    utility
    dependency
    code-sign
    build
    test
    notification
```

Reply:
Hi,

should I close this PR and open a new one or just fix tags and push (with tag change)?

Reply:
Please create a new PR with the share process, sorry for not being clear about it. Just check that there is on step.yml in the PR (that can be removed with a push if it happens)

Reply:
Not a problem :)